Automattic/wp-calypso
# Fix auto-renew cancellation form buttons

#### Changes proposed in this Pull Request

With our recent update where we moved all in-page form buttons to the left, I noticed a dialog that looked broken and had the buttons on the left (when they should be on the right. This PR fixes the button alignment for the auto-renew cancellation survey by passing the buttons directly to the dialogue vs. implementing them inline as they currently are. 

**Before**
![image](https://user-images.githubusercontent.com/6981253/97913749-c36e2f00-1d1c-11eb-85a7-145b76007b36.png)

**After**
![image](https://user-images.githubusercontent.com/6981253/97914066-48594880-1d1d-11eb-982c-c6b0c6ade1a1.png)


#### Testing instructions



* Navigate to a subscription setting screen that has an active payment method and auto-renew turned on. 
* Turn auto-renew off
* Click confirm cancellation. 
* Confirm buttons appear on the right and that you can disable auto-renew. (If you would like to see the chat, you'll need to hack `calypso/components/happychat/button` locally to show the button.
